### PR TITLE
React week-1 Homework

### PR DESCRIPTION
**Pull request by Nusrath-Jahan**
The following assignments were completed:
OurValues.js
OurCrew.js
OurPartners.js
Screenshot (83)
Screenshot (84)
Screenshot (85)

### CODE REVIEW

File: app/about_us/OurPartners.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import styles from "./partners.module.css";
+
+const OurPartners = () => {
+
+const CompanyName = [
```
Comment (vdaguenet) [inline app/about_us/OurPartners.js:5]:
```suggestion
const companyName = [
```
Capitalized variable names are reserved for components.

File: app/about_us/OurPartners.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+{company: "Alphabet",
+ Url: "./business_partners/alphabet-logo.png",
```
Comment (vdaguenet) [inline app/about_us/OurPartners.js:8]:
```suggestion
 url: "./business_partners/alphabet-logo.png",
```

File: app/about_us/OurPartners.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import styles from "./partners.module.css";
+
+const OurPartners = () => {
+
+const CompanyName = [
+
+{company: "Alphabet",
```
Comment (vdaguenet) [inline app/about_us/OurPartners.js:7]: Don't forget to indent your code

File: app/about_us/OurValues.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import styles from "./values.module.css";
+
+const OurValues = () => {
+  const value = [
+    {
+      Id: "01",
```
Comment (vdaguenet) [inline app/about_us/OurValues.js:6]:
```suggestion
      id: "01",
```

File: app/about_us/crew.module.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+.ourCrew {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 30px;
+    text-align: left;
+        
+  }
```
Comment (vdaguenet) [inline app/about_us/crew.module.css:8]: Mind the indentation

File: app/about_us/partners.module.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.ourPartners {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    margin-top: 30px;
+    justify-content: space-between;
+    text-align: justify;
+    
+  }
```
Comment (vdaguenet) [inline app/about_us/partners.module.css:9]: Mind the indentation

File: app/about_us/values.module.css [hunk 7]
```diff
@@ -0,0 +1,15 @@
+.ourValues {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    margin-top: 30px;
+    justify-content: space-between;
+    text-align: justify;
+  }
```
Comment (vdaguenet) [inline app/about_us/values.module.css:8]: Mind the indentation

**Review by vdaguenet — COMMENTED** [review]
Be careful with indentation and casing of variables.

File: app/about_us/OurPartners.js (hunk 3 above)
Comment (Nusrath-Jahan) [reply app/about_us/OurPartners.js:7]: Thank you for the recommendations.

**Pull request merged**